src/main/java/com/esri/core/geometry/ogc/OGCGeometry.java

```diff
@@ -295,10 +295,7 @@ public boolean crosses(OGCGeometry another) {
 	}
 
 	public boolean within(OGCGeometry another) {
-		com.esri.core.geometry.Geometry geom1 = getEsriGeometry();
-		com.esri.core.geometry.Geometry geom2 = another.getEsriGeometry();
-		return com.esri.core.geometry.GeometryEngine.within(geom1, geom2,
-				getEsriSpatialReference());
+		return another.contains(this);
```

Comment:
`within` is not the same as `contains`, is it? Also, why is it any different than other relationship methods? E.g. Shouldn't `com.esri.core.geometry.ogc.OGCConcreteGeometryCollection` ovewrite it?

Reply:
A.contains(B) == B.within(A), so that's why the change.

Reply:
@stolstov Makes sense. With this change `com.esri.core.geometry.GeometryEngine#within` is no longer used. Should it be removed along with `OperatorWithin` and related classes?

Reply:
Public classes and methods are likely to be used elsewhere.
e.g. https://github.com/Esri/spatial-framework-for-hadoop/blob/1ed3d421eb5c062a5309f6c610cd646210daf889/hive/src/main/java/com/esri/hadoop/hive/ST_Within.java#L20

Reply:
@mbasmanova Within/Contains, Disjoint/Intersects are a part of the API although they are trivial calls of one another. They are in OGCGeometry because they are required by the OGC spec.

Reply:
@mbasmanova The primary API for this library is the one in the core.geometry package. The core.geometry.ogc is more like the facade for the use cases that require OGC spec.

Reply:
@stolstov Sergey, thanks for clarifying.